kolibri/core/urls.py

```diff
@@ -4,12 +4,22 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from django.conf.urls import url
+from kolibri.plugins import hooks
 
 from . import views
 
 app_name = 'kolibri'
 
 urlpatterns = [
-    url('^', views.IndexView.as_view(), name='index'),
+    url('^/$', views.IndexView.as_view(), name='index'),
     # url('.* ', views.TODOView.as_view())
 ]
+
+def retrieve_plugin_urls():
```

Comment:
Happy to deprecate this in favour of whatever you and @benjaoming settle on, @MCGallaspy